lib/elixir/test/elixir/uri_test.exs

```diff
@@ -104,24 +104,24 @@ defmodule URITest do
 
   test "parse FTP" do
     assert %URI{scheme: "ftp", host: "private.ftp-servers.example.com",
-                userinfo: "user001:secretpassword", authority: "user001:secretpassword@private.ftp-servers.example.com",
-                path: "/mydirectory/myfile.txt", query: nil, fragment: nil,
+                userinfo: "user001:secret_password", authority: "user001:secret_password@private.ftp-servers.example.com",
```

Comment:
I would say passwords is alright, but i don't see why directories and filenames since underscore is what we use.